### DISCUSSION TITLE
Changes are not reflected on the homepage.

### DISCUSSION THREAD

**Discussion by SangYoupLee** [microsoft/SATechnicalOnboarding#109] [Questions]
My problem is that changes are not reflected on the hompage.

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/08fe155a-9f2f-4d89-a5f2-006c7e3dbb32)

It still has the default setting.

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/3f9867e4-8457-4d24-8e3c-4be07ec09e11)
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/df22fc5a-6569-49a7-b802-bc67c8753f9d)

Deployment was successful.. but, after 1-2minutes I got the mail that some jobs were not successful.

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/1f62e966-3895-42de-ba30-882210246a08)
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/f5485e7e-9c1c-4563-9986-30a360e528b0)

Other MLSAs said do git clone and push. Because It is not connected git repository. and then I did
do git clone and make a local repository.

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/125184499/ee1cd68a-0694-4667-b8e0-4eb4739cdef3)

Umm..

1. Making the github page by this tamplet works well.
2. BUT, deploying the webpage by Azure is not working..

I don't know how to fix this problem. By readme file, I can't finish the onboarding mission.

**Comment by AnthonyByansi** [comment]
Hi @SangYoupLee `jobs not being successful.` implies you didn't specify your output location correctly. please, have the `.github\workflow` directory deleted (You won't lose your edited work, but rather have a clean workflow) from your repository and re-deploy the app following the following steps;
![301968449-0ef7bcb2-3937-4f26-9834-38fae18c5b19](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/f8853288-97a2-4093-b3ac-8142f032f3b3)

Change the output location from the default `build` to `dist`
![301968770-0f067bff-607c-4186-b36b-834bbad46d97](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/57e2478e-56ed-4f4d-9975-636e3cb4d20a)

If you run into further issues, please DM me on Teams so I can help.

**Reply by SangYoupLee** [reply]
Thank you for the comment. I followed your suggestion. But, I faced another problem.. So.. I sent DM ON Teams about new problem. I'm sorry, but please check the message. Thank you.

**Reply by AnthonyByansi** [reply]
Alright!
